pavement.py

```diff
@@ -89,7 +89,7 @@
         GIT_REVISION = "Unknown"
 
     if not setup_py.ISRELEASED:
-        FULLVERSION += '.dev-' + GIT_REVISION[:7]
+        FULLVERSION += '.dev0+' + GIT_REVISION[:7]
```

Comment:
hm another change to the versioning?
this case is not covered by our NumpyVersion unittests